### ISSUE TITLE
parse_root_in_json regression introduced in 0.10.2

### ISSUE THREAD

**Issue by espen** [remiprev/her#508]
I have a JSON which returns a person object but my class is called member so I use parse_root_in_json :person. This has worked up to 0.10.1 but with 0.10.2 the returned object is a Member object with a person object instead of just having the person attributes on the Member object.

So instead of member.name as before, with 0.10.2 I have to do member.person.name.

Looks like https://github.com/remiprev/her/commit/df1b82b391f5fc1c53d345449338aabe809347af is the only commit in v.010.2

I'll try to put up a failing test for this.

**Comment by zacharywelch** [comment]
See the problem. I'll take a look shortly. Thanks @espen 👍

**Comment by zacharywelch** [comment]
@espen having trouble reproducing the issue. Do you see anything different from your setup and the one included in [this spec](https://github.com/remiprev/her/blob/0.x/spec/model/parse_spec.rb#L252)?

**Comment by zacharywelch** [comment]
Here's a failing spec I think captures the problem. Working on a fix...
```ruby
  context "when parse_root_in_json is set to symbol" do
    before do
      Her::API.setup url: "https://api.example.com" do |builder|
        builder.use Her::Middleware::FirstLevelParseJSON
        builder.use Faraday::Request::UrlEncoded
        builder.adapter :test do |stub|
          stub.post("/people") { [200, {}, { person: { id: 1, fullname: "Lindsay Fünke" } }.to_json] }
        end
      end

      spawn_model("Member") do
        collection_path "/people"
        parse_root_in_json :person
      end
    end

    it "parses data from root element" do
      @new_member = Member.create
      expect(@new_member.fullname).to eq("Lindsay Fünke")
    end
  end
```

**Comment by zacharywelch** [comment]
fixed by d0b49f10f19461dcb4973cdf57094069ab140b8c and published in releases [v1.0.2](https://github.com/remiprev/her/releases/tag/v1.0.2) and [v0.10.3](https://github.com/remiprev/her/releases/tag/v0.10.3). Thanks!

**Comment by edtjones** [comment]
Thanks @zacharywelch ! 😇
